### PR TITLE
Support Cloudflare provider

### PR DESCRIPTION
**Pull request by d0m0reg00dthing**
This PR adds support [Cloudflare provider](https://www.terraform.io/docs/providers/cloudflare/index.html). Currently this provider supports the following resources:

* `firewall`
  * `cloudflare_access_rule`
  * `cloudflare_filter`
  * `cloudflare_firewall_rule`
  * `cloudflare_zone_lockdown`
* `dns`
  * `cloudflare_zone`
  * `cloudflare_record`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -481,6 +483,26 @@ List of supported Datadog services:
     * `datadog_user`
 
 
+### Use with Cloudflare
+
+Example:
+```
+CLOUDFLARE_TOKEN=[CLOUDFLARE_API_TOKEN]
+CLOUDFLARE_EMAIL=[CLOUDFLARE_EMAIL]
+ ./terraformer import cloudflare --resources=firewall,dns
```
Comment (sergeylanzman) [inline README.md]: If you want, you can add token and email by flag, look github example. Not must

**Comment by sergeylanzman** [comment]
Fix for https://github.com/GoogleCloudPlatform/terraformer/issues/30

File: README.md (hunk 1 above)
Comment (d0m0reg00dthing) [reply README.md]: I thought about it and decided put creds in env vars for prettier command invocation

**Comment by d0m0reg00dthing** [comment]
> Thank you!!!
> LGTM, only need delete comment lines

Commends removed. Thank you

**Pull request merged**